hazelcast/src/test/java/com/hazelcast/query/impl/IndexTest.java

```diff
@@ -122,24 +162,30 @@ public void testIndex() throws QueryException {
         assertEquals(1, is.query(new AndPredicate(new EqualPredicate("d", "1"), new EqualPredicate("bool", false))).size());
     }
 
+    private void clearIndexes(Index ... indexes) {
```

Comment:
I would rename to clearAll.. Indexes doesn't need to be added since it is obvious from the argument type.